data-esnext.js

```diff
@@ -1719,6 +1719,8 @@ exports.tests = [
     ie11: false,
     firefox2: false,
     chrome61: chrome.harmony,
+    safari1: false,
```

Comment:
actually, this was supposed to be safari11, but i guess this is correct anyway.

Reply:
might as well update it :-)

Reply:
Actually, i'll remove this line, since I haven't actually tested it.

Reply:
Hum, I found a lot of safari1 results that I believe I added previously.  I'll remove them too.

Reply:
Should I remove previous safaritp and webkit results too?  I mean, the results refer to a previous version (not TP41).  I would obviously only remove when the results are the same as Safari 11.

Reply:
I'd say we should be keeping only results that were explicitly tested.